src/Components/Homepage/Banner/Banner.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useEffect } from "react";
+import { useNavigate } from "react-router-dom";
+import { getAllBanner } from "../../../../api/firebase";
```

Comment:
The rule of thumb is to keep the import order like this:

● Built-in
● External
● Internal